src/packages/shared-utils/package-actions/rules.ts

```diff
@@ -47,8 +47,8 @@ const arWithdrawRaiResponse: ActionRule = {
 const arWithdrawPackage: ActionRule = {
   action: Action.WITHDRAW_PACKAGE,
   check: (checker, user) =>
-    !checker.hasStatus([SEATOOL_STATUS.APPROVED, SEATOOL_STATUS.DISAPPROVED, SEATOOL_STATUS.WITHDRAWN]) &&
-    checker.isInActivePendingStatus
+    (!checker.hasStatus(finalDispositionStatuses)
+      || checker.isInActivePendingStatus)
```

Comment:
This `||` still seems redundant. We're saying "if a user doesn't have a final disposition status, OR they're in an active pending status". The former implies the latter, meaning there's no case where the second one is true if the first one is false.

I believe what we came up with was: "If a user is not in a final disposition status, they should be able to withdraw the package."